src/layouts/header.html

```diff
@@ -0,0 +1,186 @@
+<div class="header__top bg-black">
+  <div class="container mx-auto">
+    <div class="flex flex-row justify-between text-[13px]">
+      <div class="flex flex-row items-center gap-10">
+        <div class="flex flex-row items-center gap-2 hidden md:block">
+          <i class="far fa-clock text-[11px]"></i>
+          <span data-i18n="open_time">Giờ mở cửa: 8:00 - 18:00 Thứ Hai - Chủ Nhật</span>
+        </div>
+        <div class="flex-row items-center gap-5 text-[11px] hidden md:flex">
+          <a href="#" class="hover:text-white hover:bg-blue-600 w-8 h-8 flex items-center justify-center"><i
+              class="fab fa-facebook-f"></i></a>
+          <a href="#" class="hover:text-white hover:bg-blue-400 w-8 h-8 flex items-center justify-center"><i
+              class="fab fa-twitter"></i></a>
+          <a href="#" class="hover:text-white hover:bg-tumblr w-8 h-8 flex items-center justify-center"><i
+              class="fab fa-tumblr"></i></a>
+          <a href="#" class="hover:text-white hover:bg-vimeo w-8 h-8 flex items-center justify-center"><i
+              class="fab fa-vimeo-v"></i></a>
+        </div>
+        <div class="flex flex-row items-center gap-5">
+          <div class="flex items-center gap-1 cursor-pointer text-green-500" id="lang-vi">
+            <img src="../../assets/images/viet-nam.png" alt="Vietnam Flag" class="w-5 h-5">
+            <span class="hover:text-green-500">VI</span>
+          </div>
+          <div class="flex items-center gap-1 cursor-pointer" id="lang-en">
+            <img src="../../assets/images/united-kingdom.png" alt="English Flag" class="w-4 h-4">
+            <span class="hover:text-green-500">EN</span>
+          </div>
+        </div>
+      </div>
+      <div class="flex flex-row gap-5">
+        <a href="#" class="hover:text-green-500 flex items-center justify-center gap-1"><i class="fas fa-user"></i>
+          <span data-i18n="login">Đăng nhập</span></a>
```

Comment:
```suggestion
          <span data-i18n="login"></span></a>
```
Những chỗ này mình có data i18n rồi nên k cần text nữa nha, e xóa đi nhé